server/tso.go

```diff
@@ -31,9 +31,12 @@ func getTimestampPath(rootPath string) string {
 
 func (s *Server) loadTimestamp() (int64, error) {
 	data, err := getValue(s.client, getTimestampPath(s.cfg.RootPath))
-	if err != nil || data == nil {
+	if err != nil {
```

Comment:
yes, here is first check err first , if no error and data is nil, return directly. so it is ok here.